nix/sources.json

```diff
@@ -5,10 +5,10 @@
         "homepage": "",
```

Comment:
I really appreciate the version bump, but this breaks on `darwin-aarch64` 😭 I needed to roll back the `nix/` directory to get one of the `ipfs` dependencies to build

Reply:
Sure! I can totally roll back no problem. I just noticed that there was a `niv update` in the other PR and thought it wasn't a bad idea. I don't think there was another reason for me to upgrade.